templates/pages/login.jsx

```diff
@@ -53,21 +54,34 @@ var Login = React.createClass({
     // FIXME: totally not localized yet!
     var buttonText = "Log In";
     this.queryObj = Url.parse(window.location.href, true).query;
+
```

Comment:
hurray, I linked to the wrong package. It's [classnames](https://www.npmjs.com/package/classnames)